FSJ_django20_project/FSJ/views.py

```diff
@@ -205,18 +204,36 @@ def view_application(request):
         return redirect('coord_awardslist/' + str(application.award.awardid) + '/applications/')
     
     return_url = None
+    
+    context = get_standard_context(FSJ_user)
+    
     if isinstance(FSJ_user, Coordinator):
         return_url = "/coord_awardslist/" + str(application.award.awardid) + "/applications"
+        url = "/view_application?application_id=" + str(application.application_id)
+        comment_list = Comment.objects.filter(application = application)
+        context["comment_list"] = comment_list
     # TODO The return url will be whereever the adjudicator accessed the application view from, once implemented
```

Comment:
comment no longer relevant

Reply:
coordinators should no longer be able to see adjudicator comments or comments should be removed altogether?